access/access.go

```diff
@@ -33,8 +33,8 @@ func connect(engineURL string, jwtClaims jwt.MapClaims) (*enigma.Global, error)
 }
 
 // createApp creates an app using the provided Global object and app name.
-func createApp(global *enigma.Global, appName string) (*enigma.Doc, error) {
-	return global.CreateDocEx(ctx, appName, "", "", "", "Main")
+func createApp(global *enigma.Global, appName string) (bool, string, error) {
+	return global.CreateApp(ctx, appName, "Main")
```

Comment:
Couldn't you store `appId` here and return `global.OpenDoc(appId)` and have rest of the code as it was?

Reply:
Yeah that makes a lot more sense :)